protos/sources/vega/vega.proto

```diff
@@ -1295,6 +1295,14 @@ message MarketData {
   repeated LiquidityProviderSLA liquidity_provider_sla = 32;
   // Time in Unix nanoseconds when the market will next submit a trade to reduce its position.
   int64 next_network_closeout = 33;
+  // The method used for calculating the mark price.
+  CompositePriceType mark_price_type = 34;
+  // The index price used for perps.
+  optional string index_price = 35;
```

Comment:
Can we bundle the perps only bits in `optional ProductData product_data = 31;` alongside all the other perps only bits?

Reply:
we can, I think there's a caveat on how they are saved in the database (I think as json), but I think that's fine.